### DISCUSSION TITLE
Notice about fixed vulnerability missing in release notes

### DISCUSSION THREAD

**Discussion by Doctor-love** [keycloak/keycloak#17231] [Feedback]
Hi y'all,

Yesterday I was looking through the release notes for version 20.0.5 to see if there were any interesting changes or things requiring immediate attention. Based on the release notes, that didn't seem to be the case, but when I diffed 20.0.5 against 20.0.3 there seems to have been other fixes as well including what looks like a patch for a security vulnerability: https://github.com/keycloak/keycloak/pull/16764 .

Don't know if this is just a consequence of it being handled through the Github security issue workflow (that is somewhat separate from other issues) or if someone just forgot to add it.

Besides a potential heads-up about this, I just wanna straighten out if this was done on purpose or not. Personally I use the release notes to determine if I need to prioritize applying the updates ahead of scheduled maintenance windows or not - a process that I think many other admins follow as well.

Thanks for the release and keep up the great work - Keycloak keeps getting better and better!

**Comment by Doctor-love** [comment]
I see the same issue with the latest release - no reference is made to to security issues fixen in https://github.com/keycloak/keycloak/commit/4ffb69ecefce155f297d3bb9f6ecc8fa8600d308

Just based on the release date and similarities in description, I guess that the following CVEs are related:

https://www.cvedetails.com/cve/CVE-2024-2419/
https://www.cvedetails.com/cve/CVE-2024-1249/
https://www.cvedetails.com/cve/CVE-2024-1132/

I highly encourage the project to look at home they handle communication vulnerabilities and releases.

**Reply by Doctor-love** [reply]
Ping @stianst

**Comment by stianst** [comment]
We're aware of this and will be improving on it. Not sure when though as it requires a bit of smart thinking to have the list of vulnerabilities available when we're generating the release notes.

**Reply by abstractj** [reply]
@Doctor-love meanwhile you should be able to check the list of most recent published CVEs here https://github.com/advisories?query=keycloak%2Fkeycloak.
